icevision/core/record_components.py

```diff
@@ -275,8 +276,45 @@ def setup_transform(self, tfm) -> None:
 
 
 class ClassificationLabelsRecordComponent(BaseLabelsRecordComponent):
+    is_multilabel = False
+
+    def __init__(self, task=tasks.classification):
+        super().__init__(task=task)
+
+
+class MultiLabelClassificationLabelsRecordComponent(BaseLabelsRecordComponent):
+    """Component for multi-label classification
+    * Doesn't allow you to `add_labels`
+    * One-hot encodes labels based on `class_map`
+    """
+
+    is_multilabel = True
```

Comment:
`is_multilabel` is added for the ability to recognise the component during data loading time.

We could get rid of this and instead do
```python
if composite.get_component_by_type(MultiLabelClassificationLabelsRecordComponent)
```